src/components/inputs/CopyInput/CopyInput.tsx

```diff
@@ -74,7 +74,7 @@ export const CopyInput = (props: ICopyInputProps) => {
 			>
 				<Tooltip
 					{...inputTooltipProps}
-					hideTooltip={inputTooltipProps ? inputTooltipProps.hideTooltip : true}
+					hideTooltip={inputTooltipProps ? inputTooltipProps?.hideTooltip : true}
```

Comment:
Ah, my bad! I meant the first instance, like so:

`hideTooltip={inputTooltipProps?.hideTooltip ? inputTooltipProps?.hideTooltip : true}`

Reply:
Oh yeah I see, my change did nothing haha. But I think that changes the intent a little - I want the tooltip to default to hidden when no TooltipProps are passed, and default to showing when they are passed but `hideTooltip` isn't. I think if `hideTooltip` isn't defined, it'll be falsey

Reply:
Could change it to:

`hideTooltip={inputTooltipProps ? (inputTooltipProps.hideTooltip ?? false) : true}`

but I think that's redundant since undefined is falsey

Reply:
Ah right! With undefined resolving to falsey then we should be good. Approving!